library/globals.inc.php

```diff
@@ -288,6 +288,8 @@ function gblTimeZones()
             array(
                 'dual' => xl('Dual'),
                 'comprehensive' => xl('Comprehensive'),
+                'fixed' => xl('fixed'),
```

Comment:
Capitalize Fixed and capitalize None

Reply:
For cap, I only mean the labels that are being translated.

Reply:
Of course thanks